ooiui/static/js/models/common/PageControlModels.js

```diff
@@ -25,43 +25,62 @@ var TocPageControlModel = Backbone.Model.extend({
 
         params : {
             "General":[
-                ["temperature", "Temperature"],
-                ["pressure", "Pressure"],
-                ['ph', 'pH'],
-                ['nitrate', 'Nitrate'],
-                ["co2", "CO2"],
-                ['density', 'Density'],
-                ['salinity', 'Salinity']
+                ["CTDAVN CTDBPC CTDBPD CTDBPE CTDBPF CTDBPN CTDBPO CTDBPP CTDGVM CTDMOG CTDMOH CTDMOQ CTDMOR CTDPFA CTDPFB CTDPFJ CTDPFK CTDPFL", "Water Temperature"],
+                ["CTDAVN CTDBPC CTDBPD CTDBPE CTDBPF CTDBPN CTDBPO CTDBPP CTDGVM CTDMOG CTDMOH CTDMOQ CTDMOR CTDPFA CTDPFB CTDPFJ CTDPFK CTDPFL", "Pressure"],
+                ['PHSENA PHSENB PHSEND PHSENE PHSENF', 'pH'],
+                ['NUTNRB NUTNRA NUTNRM', 'Nitrate'],
+                ['CTDAVN CTDBPC CTDBPD CTDBPE CTDBPF CTDBPN CTDBPO CTDBPP CTDGVM CTDMOG CTDMOH CTDMOQ CTDMOR CTDPFA CTDPFB CTDPFJ CTDPFK CTDPFL', 'Density'],
+                ['CTDAVN CTDBPC CTDBPD CTDBPE CTDBPF CTDBPN CTDBPO CTDBPP CTDGVM CTDMOG CTDMOH CTDMOQ CTDMOR CTDPFA CTDPFB CTDPFJ CTDPFK CTDPFL', 'Salinity']
```

Comment:
Add  ['OPTAAC OPTAAD OPTAAJ', 'Chlorophyll'],
Then add 'FLNTUA FLCDRA FLORDD FLORDL FLORDM FLORTD FLORTJ FLORTK FLORTM' to the chlorophyl list

Add +                ['OPTAAC OPTAAD OPTAAJ', 'Turbidity'],